tests/netlify-deploy.ts

```diff
@@ -148,7 +148,7 @@ export class NextDeployInstance extends NextInstance {
 
     try {
       const deployUrlRegex = new RegExp(
-        /https:\/\/app\.netlify\.com\/sites\/(?<siteName>[^\/]+)\/deploys\/(?<deployID>[0-9a-f]+)/gm,
+        /https:\/\/app\.netlify\.com\/(sites|projects)\/(?<siteName>[^\/]+)\/deploys\/(?<deployID>[0-9a-f]+)/gm,
```

Comment:
it's `projects` now, but kept old one in there in case there would be any kind of revert on that